MaterialLib/FractureModels/FractureIdentity2.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef MATERIALLIB_FRACTURE_IDENTITY2_H_
+#define MATERIALLIB_FRACTURE_IDENTITY2_H_
+
+#include <Eigen/Eigen>
+
+namespace MaterialLib
+{
+namespace Fracture
+{
+
+template <int DisplacementDim>
+struct FractureIdentity2
```

Comment:
this corresponds to `Invariants::identity2` in KelvinVector.h for usual solids. I will add some comment there